docs/api/setupI13n.md

```diff
@@ -27,6 +27,15 @@ var I13nDempApp = setupI13n(DemoApp, {
 }, [someReactI13nPlugin]);
 
```

Comment:
looks like you are missing closing back ticks for the example. This messes up formatting below.